common/app/model/dotcomrendering/InteractiveSwitchOver.scala

```diff
@@ -0,0 +1,7 @@
+package model.dotcomrendering
+
+import org.joda.time.DateTime
+
+object InteractiveSwitchOver {
+  val date = DateTime.parse("2021-06-21T00:00Z")
```

Comment:
I think it's probably worth commenting this to explain the context, even though the naming is pretty clear here already.

Reply:
👍